src/main/java/com/example/demo/Category/Category.java

```diff
@@ -0,0 +1,24 @@
+package com.example.demo.Category;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table
+@Getter
+@Setter
+public class Category {
+
+    private Integer id;
+    private  String name;
```

Comment:
Extra space, install some kind of plugin to intellij to auto format java code

Reply:
done